docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,67 +14,80 @@ endif::[]
+The difference between the two methods is in how exactly that certificate gets to the vehicle, and who controls the Fleet Root CA. Let's take a closer look at how each method works.
 
-Let's take a close look at how each method works.
 
 == Provisioning with shared credentials
 
-This method is called "provisioning with shared credentials" because you install a temporary provisioning key that is shared by all devices.
-With this method, perform the following major steps:
+This type of provisioning is the default, and is great to start with, because the OTA Connect server does everything for you: it generates a Fleet Root CA for your account, and it generates a new vehicle certificate every time a new device comes online. This allows you to try out OTA Connect without involving the rest of your infrastructure. However, this method may not be secure enough for a production scenario.
 
-* You download a temporary provisioning key from the OTA Connect server and install it on a base software image.
-* You then flash your base software image to your devices.
-* Once each device boots up, it uses the shared provisioning key to request a permanent device certificate from the OTA Connect server.
-* The OTA Connect server verifies the provisioning key, issues the device with an X.509 certificate which is then downloaded to the device.
-* The device then uses this certificate for all future transactions.
+In shared credential provisioning, the OTA Connect server creates the fleet root CA for you, and stores the private key for the CA in a https://www.vaultproject.io/[Vault] instance. The server also generates a credential for you that can be shared amongst all the devices in your fleet. That credential--also known as a provisioning key--is included in your `credentials.zip`, and is baked into the generic image that you flash on all your devices.
+
+In this method, the following steps occur:
+
+. You download your `credentials.zip` with the provisioning key inside.
+. You bitbake an image with the default provisioning method configured, providing bitbake with your `credentials.zip`.
+. You flash that image on as many devices as you wish.
+. The first time each device comes online, it connects to a special endpoint on the device gateway and says, “Hi, I’m a new device, and I don’t have a TLS certificate yet. Here’s my provisioning key.”
+. Our crypto service generates a new keypair and X.509 certificate for the device, signs that certificate with the private Fleet Root CA in the Vault instance, and sends the whole bundle to the vehicle.
+. The crypto service deletes the vehicle’s private key and the device stores its new keypair and certificate.
+
+This entire transaction is secured with TLS, using the pinned device gateway certificate provisioned in the image.
+
+.Summary of Shared credential provisioning
+[caption="Figure 1: "]
+image::shared-cred-provisioning.png[1280, 500]
```

Comment:
Where is this image? It doesn't show up in github, which might not matter.

Reply:
Yup, this does seem to be missing. @ammaboateng ?

Reply:
@tkfu should I send you the image files to be added? I think they were lost in one of the commits.